templates/repo/settings/options.tmpl

```diff
@@ -1012,6 +1019,30 @@
 		</div>
 	</div>
 
+	{{if not .Repository.IsFork}}
+		<div class="ui g-modal-confirm modal" id="visibility-repo-modal">
+			<div class="header">
+				{{ctx.Locale.Tr "repo.visibility"}}
+			</div>
+			<div class="content">
+				<p>
```

Comment:
Is this `<p>` needed or at least consistent?

Reply:
To keep consistency the same as in the [archive modal](https://github.com/go-gitea/gitea/blob/main/templates/repo/settings/options.tmpl#L1058-L1064), but if needless I can remove it.

Reply:
Fine if it's consistent.